<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMemberPopUp</class>
 <widget class="QDialog" name="AddMemberPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Basic Member</string>
  </property>
  <widget class="QLabel" name="enterMemberNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter member name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterMemberNameLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectTypeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select membership type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="selectTypeComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Preferred</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Basic</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>168</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addMemberPushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelAddMemberPushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="enterMemberExpirationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter membership expiration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addMemberInvalidEntryLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Invalid  entry</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="enterMemberExpirationMonthLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="slashLabel_1">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="enterMemberExpirationDayLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="slashLabel_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="enterMemberExpirationYearLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="enterMemberExpirationFormatLabel">
      <property name="text">
       <string>(MM/DD/YYYY)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enterMemberNameLineEdit</tabstop>
  <tabstop>selectTypeComboBox</tabstop>
  <tabstop>enterMemberExpirationMonthLineEdit</tabstop>
  <tabstop>enterMemberExpirationDayLineEdit</tabstop>
  <tabstop>enterMemberExpirationYearLineEdit</tabstop>
  <tabstop>addMemberPushButton</tabstop>
  <tabstop>cancelAddMemberPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
